deliverylocation/tests/test_deliverylocation.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+from deliverylocation.models import Supplier, DeliveryLocation
+from django.core.exceptions import ValidationError
+import datetime
+
+test_data = [
+                ('taxas', 'firstname', 'lastname', 'password', 'supplier', 'business name'),
+                ('ed', 'edd', 'eddy', 'jawbreaker101', 'supplier', 'jawbreaker inc'),
+        ]
+
+
+@pytest.fixture
+def supplier0():
+    return Supplier(test_data[0])
+
+
+@pytest.fixture
+def deliverylocation0(supplier0):
```

Comment:
I'd prefer it in snake_case. Same goes to everything in this file which is function or variable name.

Reply:
Done